src/clipboard.js

```diff
@@ -35,11 +42,24 @@ class Clipboard extends Emitter {
      * Delegates a click event on the passed selector.
      * @param {String} selector
      */
-    delegateClick(selector) {
+    delegateClickToSelector(selector) {
         Delegate.bind(document.body, selector, 'click', (e) => this.initialize(e));
     }
 
     /**
+     * Delegates a click event on the passed element.
+     * @param {Object} element
+     */
+    delegateClickToElement(element) {
+        event.bind(document.body, 'click', (e) => {
+            e.delegateTarget = element;
+            if (e.delegateTarget) {
+                (e) => this.initialize(e);
```

Comment:
Why the anonymous function? Shouldn't be `this.initialize(e)` only? Or `this.onClick(e)` after 1.4 / cc9d562580efb3d0a33ecd5b8a83ed6ae9d884cf.